database/index.js

```diff
@@ -114,21 +114,16 @@ var saveUser = (user) => {
   })
 }
 
-// var selectImages = (callback, query) => {
-//   Image.find({query}, (err, results) => {
-//     if (err) {
-//       callback(err, null);
-//     } else {
-//       callback(null, results);
-//     }
-//   });
-// }
-
-// var save = () => {
-//
-// }
+
+
+var update = (imageData) => {
+  Image.update({ _id: imageData.data._id }, { $set: { likeCount: imageData.data.likeCount, dislikeCount: imageData.data.dislikeCount, comments: imageData.data.comments }}, () => {
```

Comment:
note to refactor later,  since we will be sending req.body.data we can remove the .data part on all the references.  cleans it up a bit.